draft.md

```diff
@@ -77,6 +77,10 @@ This week’s release was curated by [Maëlle Salmon](https://twitter.com/ma_sal
 
 + [{epoxy}: String Interpolation for 'R Markdown' Documents ](https://github.com/gadenbuie/epoxy)
 
++ [{spikemap} : Maps Made of Spikes](https://github.com/rCarto/spikemap)
+
+<img src = "https://raw.githubusercontent.com/rCarto/spikemap/master/img/spiky.png" width = "500px" align = "center">
```

Comment:
Please remove the image from the PR. The editor in charge of the release (me) might add it, but that'll happen later and with a different format.

Reply:
(and thanks for the PR!!)

Reply:
Sure, no problem!